bin/create_roles.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+##This script will create the user roles.
```

Comment:
could we rename this script like create-initial-roles, please?

Reply:
Renamed the script.